client/components/SocialClub/SocialClubPage.js

```diff
@@ -190,32 +165,32 @@ class SocialClubPage extends Component {
 
                                 <div className="form-group">
                                   <label  htmlFor="title">title</label>
-                                  <input value={this.state.title}  onChange={this.onChange} type="" className="form-control" id="title" placeholder="title" name="title"/>
+                                  <input value={this.state.newEvent.title}  onChange={this.onChange} type="" className="form-control" id="title" placeholder="title" name="title"/>
```

Comment:
Let's add `type="text"` to our input tags, so we can know what each of them is used for.

Reply:
Ok... noted